entoas/annotation.go

```diff
@@ -50,6 +52,10 @@ type (
 	}
 	// OperationConfigOption allows managing OperationConfig using functional arguments.
 	OperationConfigOption func(*OperationConfig)
+	// FieldConfig hold meta information aboat a REST field (parameter)
+	FieldConfig struct {
+		ReadOnly bool
+	}
```

Comment:
I think we can drop this and add the `ReadOnly` field directly on the `Annotation`.

Reply:
Done. 1. ReadOnly directly in Annotation, 2. Builder added, 3. Unit tests added

Once merged, I'll update the corresponding ogent PR